packages/documentation/src/pages/forms/using-available-fields.md

```diff
@@ -131,3 +132,24 @@ properties: {
   },
 },
 ```
+
+
+## Misc Field options
+
+### `addAnotherLabel`
+
+- Only applies when setting a schema type to `array`.
+- The `ArrayField` code will add an "Add Another" button with a suffix from `itemName` (usually in title-case)
```

Comment:
```suggestion
- The `ArrayField` code will add an "Add another" button with a suffix from `itemName`.
```
When the global change is made, it will be sentence-cased.